### PR TITLE
Bugfix: pad is max(width,const) during constprop

### PR DESCRIPTION
**Pull request by azidar**

### CODE REVIEW

**Comment by aswaterman** [comment]
Ah, right.

Prefer a max b, rather than math.max(a, b).

File: src/main/scala/firrtl/passes/ConstProp.scala [hunk 1]
```diff
@@ -198,8 +198,8 @@ object ConstProp extends Pass {
       case _ => e
     }
     case Pad => e.args.head match {
-      case UIntLiteral(v, _) => UIntLiteral(v, IntWidth(e.consts.head))
-      case SIntLiteral(v, _) => SIntLiteral(v, IntWidth(e.consts.head))
+      case UIntLiteral(v, IntWidth(w)) => UIntLiteral(v, IntWidth(max(e.consts.head, w)))
+      case SIntLiteral(v, IntWidth(w)) => SIntLiteral(v, IntWidth(max(e.consts.head, w)))
```
Comment (jackkoenig) [inline src/main/scala/firrtl/passes/ConstProp.scala]: change to `e.consts.head max w`

**Comment by azidar** [comment]
Lol not sure what happened with the commit history, but the last commit is the update. @jackkoenig, please review.

**Comment by jackkoenig** [comment]
This looks like the commit history got all messed up. @azidar can you fix it?

**Comment by azidar** [comment]
subsumed by #465

**Pull request closed without merging**